frontend/src/components/AppFooter.vue

```diff
@@ -37,14 +37,14 @@ export default {
         { content: 'Profile', href: '/profile' }
       ],
       info: [
-        { content: 'Privacy Policy', href: '/' },
-        { content: 'Terms of Use', href: '/' },
-        { content: 'Help', href: '/' },
-        { content: 'UNSW Handbook', href: '/' }
+        { content: 'Privacy Policy', href: '/privacy-policy' },
+        { content: 'Terms of Use', href: '/terms-of-service' },
+        { content: 'Help', href: '/', disabled: 'cursor: not-allowed;' },
+        { content: 'UNSW Handbook', href: '/handbook' }
       ],
       social: [
-        { content: 'facebook', href: '/' },
-        { content: 'twitter', href: '/' }
+        { content: 'facebook', href: '/', disabled: 'cursor: not-allowed;' },
```

Comment:
link facebook to CSESoc.

Reply:
Add a Contribute link which points to the repo.

Reply:
imo don't link to CSEsoc lol... contribute link is a good idea